<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LJQcListWin</class>
 <widget class="QWidget" name="LJQcListWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LJQcListWin</string>
  </property>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbExpDate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exp. Date</string>
   </property>
  </widget>
  <widget class="LineEdit" name="leLotNo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="DateLineEdit" name="leExpDate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbLotNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot No.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDelete">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPageUp">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPageDown">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrint">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DateLineEdit</class>
   <extends>LineEdit</extends>
   <header>datelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
